install_files/ansible-base/roles/backup/files/0.3_collect.py

```diff
@@ -56,7 +56,8 @@ def encrypt_zip_file(zf_fn):
     gpg = gnupg.GPG(binary='gpg2', homedir=config.GPG_KEY_DIR)
     e_fn = '{}.gpg'.format(zf_fn)
 
-    gpg.encrypt_file(zf_fn, config.JOURNALIST_KEY, always_trust='True', output=e_fn)
+    stream = open(zf_fn, "rb")
+    gpg.encrypt_file(stream, config.JOURNALIST_KEY, always_trust='True', output=e_fn)
```

Comment:
I thought it was suppose to too but, when I didn't open it first and left it to encrypt_file to handle that would get this error trying to unzip the decrypted file:
``` 
me@me:~/git/securedrop$ gpg -o test.zip -d install_files/ansible-base/sd-backup-2014-12-17--03-03-17.zip.gpg 
gpg: encrypted with 4096-bit RSA key, ID A2201B2A, created 2013-10-12
      "SecureDrop Test/Development (DO NOT USE IN PRODUCTION)"
me@me:~/git/securedrop$ unzip test.zip 
Archive:  test.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of test.zip or
        test.zip.zip, and cannot find test.zip.ZIP, period.
```